random-beans/src/main/java/io/github/benas/randombeans/randomizers/collection/MapRandomizer.java

```diff
@@ -49,16 +49,6 @@
     private final Randomizer<V> valueRandomizer;
 
     /**
-     * Create a new {@link MapRandomizer} with a random number of entries.
-     *
-     * @param keyRandomizer   the randomizer for keys
-     * @param valueRandomizer the randomizer for values
-     */
-    public MapRandomizer(final Randomizer<K> keyRandomizer, final Randomizer<V> valueRandomizer) {
```

Comment:
I used my IDE to remove all the dead code - if this is used, what could likely be true, i would write a test for it and readd it